frontend/src/modules/app/ToolBar.tsx

```diff
@@ -74,18 +80,35 @@ interface AlarmConfiguration {
   stateKey: string;
```

Comment:
@rohanpurohit Did you copy-paste this code from `modules/alarms/AlarmsPage.tsx`? If so, it'd be better just import the interface from there instead, for maintainability & commonality.

Reply:
This has been fixed in #362